types/MimicryNFTMarketIndex.md

```diff
@@ -0,0 +1,205 @@
+## Type Name
+
+- `MimicryNFTMarketIndex`
+
+
+## Description
+
+This query returns a market capitalization metric derived from the 50 most valuable NFT collections' sales data on a particular chain. 
+
+Its initial use case is enabling the creation of a macro-market on [Mimicry Protocol](https://www.mimicry.org/), but the values can be used by anyone who needs this metric to infer the directional market sentiment of traders of NFTs.
+
+### Market Capitalization, Explained
+Market capitalization is calculated as the sum of each NFT within the 50 most valuable NFT collections, where each NFT is valued at the greater of its last traded price or the floor price of the collection, respectively.
+
+#### For ERC721 collections:
+- Market Cap = the sum of all NFT values of the collection
+- The value of each NFT = max(last price, floor price)
+
+#### For ERC1155 collections:
+- Market Cap = the sum of all NFT values of the collection
+- The value of each NFT = max(last price, floor price) times the number of owners
+
+Suspected wash trades should be filtered out.
+
+> Note that this market capitalization methodology has been heavily influenced by NFTGo's [collection metrics documentation](https://docs.nftgo.io/docs/collection-metrics). A visualization of the top 50 most valuable collections, according to NFTGo, is available [here](https://nftgo.io/discover/top-collections?timeSpan=all&sort=marketCapEth). 
+
+
+## Query Parameters
+
+The `MimicryNFTMarketIndex` query has two parameters:
+
+1. **chainId**
+    - description: The chain id of the blockchain that the collections live on
+    - value type: `uint256`
+    - allowed values:
+        - `0` - ETH Mainnet
+        - `1` - Solana Mainnet
```

Comment:
I would also make chain a string, or if you're only using EVM's , use a number.  The issue is that chainID is an actual variable in EVM blockchains where 1 is ETH Mainnet, so this is super confusing.

Reply:
Roger that. Just made the change. The thinking on my end with IDs was that reporters will probably code up a switch statement (or something similar) and only support a small number of chains to start. Easy enough for them to switch against a string though.